ros2_batch_job/__main__.py

```diff
@@ -45,14 +45,15 @@
 sys.stderr = UnbufferedIO(sys.stderr)
 
 pip_dependencies = [
+    'EmPy',
```

Comment:
yep you're right that that's how the 'google-style' for flake8-import-order does it. I think it might be more common to sort considering case, so I'll leave it like this unless there are objections